esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -389,26 +431,41 @@ def __init__(
             exceeds ``mdtol``. ``mdtol=0`` means no missing data is tolerated while
             ``mdtol=1`` will mean the resulting element will be masked if and only
             if all the contributing elements of data are masked.
+        method : str, default="conservative"
+            The regridding method which :mod:`ESMF` uses to calculate weights. Can be
+            either "conservative" or "bilinear".
```

Comment:
Could reference the original class attributes, to make this less of a closed box, e.g. 

```
:attr:`ESMF.RegridMethod.CONSERVE`
```

Or is that overkill?

Reply:
Just tested this and the correct path is actually

```
:attr:`ESMF.api.constants.RegridMethod.CONSERVE`
```

So maybe `~ESMF...` is called for.

Sorry for missing that!